README.md

```diff
@@ -193,7 +193,7 @@ KURS_INFO_API_KEY=[key]
 
 SESSION_SECRET=[secret]
 SESSION_KEY=[key]
-APPINSIGHTS_INSTRUMENTATIONKEY=[key]
+APPLICATIONINSIGHTS_CONNECTION_STRING=[connection string]...
```

Comment:
Why `...` ?

Reply:
Hm, good question! I don't know, some kind of typo.